src/entitysdk/util.py

```diff
@@ -96,10 +96,9 @@ def stream_paginated_request(
             token=token,
             http_client=http_client,
         )
-        json_data = response.json()["data"]
-
-        if not json_data:
```

Comment:
Is there any possibility to receive a non-error response that doesn't have a data key?

Reply:
It should be possible only in case of error, because. the response model is defined as 
https://github.com/openbraininstitute/entitycore/blob/250d6c02effa1d576d5a2550d597c1a8b90b2676/app/schemas/types.py#L38-L41

Reply:
ok, should be good then without it.